docs/release-notes/release-notes-0.16.0.md

```diff
@@ -170,7 +170,10 @@ certain large transactions](https://github.com/lightningnetwork/lnd/pull/7100).
 
 * [A new config option, `mailboxdeliverytimeout` has been added to
   `htlcswitch`](https://github.com/lightningnetwork/lnd/pull/7066).
-
+  
+* [Remove accidentally copied "lightning:" prefix from
+  `pay_req` flag](https://github.com/lightningnetwork/lnd/pull/7150)
```

Comment:
Nit: make release notes more user-friendly?

The `pay_req` field for `sendpayment` and `decodepayreq` may optionally begin with the `lightning:` prefix often used by mobile applications.